packages/create-cloudflare/templates/do-hello-world/js/src/index.js

```diff
@@ -0,0 +1,29 @@
+/**
+ * Welcome to Cloudflare Workers! This is your first Durable Objects application.
+ *
+ * - Run `npm run dev` in your terminal to start a development server
+ * - Open a browser tab at http://localhost:8787/ to see your Durable Object in action
+ * - Run `npm run deploy` to publish your application
+ *
+ * Learn more at https://developers.cloudflare.com/durable-objects
+ */
+
+export class MyDurableObject {
+	constructor(state, env) {}
+
+	async fetch(request) {
+		return new Response("Hello World");
+	}
+}
+
+export default {
+	async fetch(request, env, ctx) {
+		let id = env.MY_DURABLE_OBJECT.idFromName(new URL(request.url).pathname);
+
+		let stub = env.MY_DURABLE_OBJECT.get(id);
+	
+		let response = await stub.fetch(request);
```

Comment:
It might be worth a comment clarifying that we need to get a stub to communicate with the DO?

```
// This stub creates a communication channel with the Durable Object instance.
// When we call `fetch()` on the stub, it sends the request to the Durable Object instance 
// and runs its fetch handler.
let stub = env.MY_DURABLE_OBJECT.get(id);
let response = await stub.fetch(request);
```